editor/import/resource_importer_scene.cpp

```diff
@@ -2077,7 +2078,14 @@ void ResourceImporterScene::_generate_meshes(Node *p_node, const Dictionary &p_m
 					src_mesh_node->get_mesh()->create_shadow_mesh();
 				}
 
-				if (!save_to_file.is_empty()) {
+				String save_path = save_to_file.get_base_dir();
+				bool save_path_exists = DirAccess::exists(save_path);
+
+				if (!save_to_file.is_empty() && !save_path_exists) {
+					WARN_PRINT("Mesh save path '" + save_path + "' doesn't exist.");
+				}
```

Comment:
Is this an if.. else pattern?

Reply:
It's not - the warning check has to be independent from the rest of the pattern since the last 'else' needs to be triggered in order for the mesh to be saved in the scene, otherwise it doesn't get saved at all.